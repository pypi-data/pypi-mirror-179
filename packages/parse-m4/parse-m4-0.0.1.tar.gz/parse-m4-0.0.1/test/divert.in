this is the beginning
divert(2)
this is 2
divert(1)
this is 1
divert(100)
this is 100
divert(3)
this is 3
divert(-1)
this should not appear
nor should this
divert
this is the end
