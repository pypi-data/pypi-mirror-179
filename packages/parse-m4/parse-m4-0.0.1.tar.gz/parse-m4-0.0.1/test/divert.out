this is the beginning

this is the end

this is 1

this is 2

this is 3

this is 100
